py-scoring/README.rst

```diff
@@ -7,6 +7,7 @@ This package contains just functionality for scoring with Sparkling Water, H2O-3
 
 Documentation describing scoring with H2O-3 MOJO models is located at:
 
+- For Spark 3.5 - https://docs.h2o.ai/sparkling-water/3.5/latest-stable/doc/deployment/load_mojo.html
```

Comment:
Shouldn't the changes in this file be rather a part of https://github.com/h2oai/sparkling-water/pull/5664?